install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -193,7 +193,11 @@ spec:
 {{- with .Values.extraEnv }}
 {{ toYaml . | trim | indent 8 }}
 {{- end }}
+{{- if .Values.image.useDigest }}
```

Comment:
any reason not to use .Values.image.digest here directly (i.e. check if .Values.image.digest is not empty) ?

Reply:
@sayboras see https://github.com/cilium/cilium/pull/15185#discussion_r586700040 if it makes sense